app/services/saml_request_validator.rb

```diff
@@ -75,6 +75,6 @@ def email_nameid_format?
   end
 
   def service_provider_allowed_to_use_email_nameid_format?
-    Saml::Idp::Constants::ISSUERS_WITH_EMAIL_NAMEID_FORMAT.include?(service_provider.issuer)
+    service_provider.allow_email_nameid_format
   end
```

Comment:
ditto, what if we removed this method and moved things up?

Reply:
done!